server/chat-commands/core.ts

```diff
@@ -965,6 +965,24 @@ export const commands: ChatCommands = {
 	},
 	importinputloghelp: [`/importinputlog [inputlog] - Starts a battle with a given inputlog. Requires: + % @ &`],
 
+	async reseedbattle(target, room, user) {
```

Comment:
I said before, but this should be part of the `/editbattle` command, not its own command.